src/OpenTelemetry.Api/Trace/SpanHelper.cs

```diff
@@ -71,43 +57,19 @@ public static string GetCachedCanonicalCodeString(StatusCanonicalCode statusCano
         /// <returns>Resolved span <see cref="Status"/> for the Http status code.</returns>
         public static Status ResolveSpanStatusForHttpStatusCode(int httpStatusCode)
         {
-            var newStatus = Status.Unknown;
+            var newStatus = Status.Unset;
 
             if (httpStatusCode >= 100 && httpStatusCode <= 399)
             {
                 newStatus = Status.Ok;
             }
-            else if (httpStatusCode == 400)
+            else if ((httpStatusCode >= 400 && httpStatusCode <= 404)
```

Comment:
This section can fold to `httpStatusCode >= 400`.

Reply:
I think the logic here should be:
```csharp
var status = Status.Error;
if (httpStatusCode >= 100 && httpStatusCode <= 399)
{
    status = Status.Ok;
}

Reply:
thanks for the suggestion

Reply:
Ahh.. I remembered why i didn't do this...
Imagine that we receive a different result code, for example, 999.
Since that is an unknow i was setting to `Unset` instead of `Error`. What do you think?

Reply:
999 is an error since it is unexpected.